conda_devenv/devenv.py

```diff
@@ -1,31 +1,40 @@
+# mypy: disallow-untyped-defs
```

Comment:
If everything is now annotated, maybe these configs can be moved to `mypy.ini`?

Reply:
Good idea, done!